packager/hls/base/media_playlist.cc

```diff
@@ -170,7 +174,9 @@ class SegmentInfoEntry : public HlsEntry {
                    bool use_byte_range,
                    uint64_t start_byte_offset,
                    uint64_t segment_file_size,
-                   uint64_t previous_segment_end_offset);
+                   uint64_t previous_segment_end_offset,
+                   bool use_program_date_time,
+                   double start_timestamp);
```

Comment:
Can you rename |start_timestamp| to |program_date_time| to avoid confusion as we already have a start_time in this class?

Reply:
Done.